lib/hanami/cli.rb

```diff
@@ -131,6 +131,16 @@ def routes
       end
     end
 
+    desc 'secret', 'Prints a secure hash'
+    long_desc <<-EOS
+    `hanami secret` will generate a secure key to be used by the developer as he wants, e.g. cookie sessions.
+     The secret will have the desired length * 2.
```

Comment:
I'd personally prefer `hanami secret 20` to output a 20 character string, rather than a 40 character one.

If we want to keep it the same as how `SecureRandom` works, then can we at least explain that they're specifying the number of bytes, rather than number of characters?